package.json

```diff
@@ -82,6 +82,7 @@
   "typeCoverage": {
     "atLeast": 100,
     "detail": true,
+    "ignoreNested": true,
```

Comment:
I added this, because `InitializeRequest.type` is of type `ProtocolRequestType<_InitializeParams & WorkspaceFoldersInitializeParams & WorkDoneProgressParams, InitializeResult<any>, never, InitializeError, void>`.

In other words: An upstream type includes an `any` type, which we don’t even use.

Reply:
Oh interesting. I didn’t know about this field. It sounds like it might be useful in other projects too